### ISSUE TITLE
error analyzing component.js

### ISSUE THREAD

**Issue by benxshen** [ipatalas/ngComponentUtility#1]
Hi team,

This plugin fails to analyze my component files. Here is one of my components source:

popGoogleLoginButton.component.js:
```js
(function () {
   'use strict';

    // Usage:
    //
    // Creates:
    //

    angular
        .module('PollexOneApp')
        .component('popGoogleLoginButton', {
            template: '<button type="button" class="google-login-button btn "><span><img class="google-img" src="assets/img/login/google.png" ></img></span>&nbsp;<span translate="login.title">Sign In</span></button>',
            controller: GoogleLoginButtonController
        });

    /* @ngInject */
    function GoogleLoginButtonController($element, GoogleAuth) {
        var $ctrl = this;

        ////////////////

        $ctrl.$onInit = function () {
            $element.hide();
        };

        $ctrl.$postLink = function() {

            GoogleAuth.init().then(function(){
                GoogleAuth.attachLoginAction($element.get(0), function onSignIn() {
                    $element.find('.google-login-button').attr('disabled', true);
                });
                $element.show();
            });

        };

        ////////////////
    }
})();

```

**Comment by ipatalas** [comment]
Hello,

thanks for reporting. As said in the readme file we were focused mainly on TypeScript based components as this is what we use in our project. However I am gonna look at this issue as it seems it should have been analyzed properly. Thanks for example file, it will make analysis so much easier.

Happy New Year :)

**Comment by ipatalas** [comment]
Fixed in 0.3.2
